core/storage/feedback/gae_models.py

```diff
@@ -287,3 +287,15 @@ def get_by_exploration_and_thread_id(cls, exploration_id, thread_id):
         Returns None if it doesn't match anything."""
 
         return cls.get_by_id(cls._get_instance_id(exploration_id, thread_id))
+
+
+class FeedbackMessageEmailDataModel(base_models.BaseModel):
```

Comment:
The name you suggest fits perfect for its context. I think its unnecessarily too long (and uneasy to understand). Current name of model is also fine according to me.
So if you prefer new name then I will do changes ?